src/sentry/search/events/constants.py

```diff
@@ -295,6 +295,8 @@ class ThresholdDict(TypedDict):
     "http.response_content_length": "d:spans/http.response_content_length@byte",
     "http.decoded_response_content_length": "d:spans/http.decoded_response_content_length@byte",
     "http.response_transfer_size": "d:spans/http.response_transfer_size@byte",
+    # Duration is only collected for specific app start spans
+    "duration": "d:spans/duration@millisecond",
```

Comment:
shouldn't be necessary 👀 https://github.com/getsentry/sentry/pull/63144/files#diff-5ef5f3794ce0417331dd489ccb931ca76a275f37a470840ade5fc482999a57e7R294

Reply:
Ughh lol yeah totally right. I just looked up.

When I was doing the frontend I slapped `duration` into my field and forgot to prefix it with `span.duration` and just assumed it was removed before. Closing this PR, thanks 😅